index.bs

```diff
@@ -1260,6 +1263,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
             1. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation
                 on |authenticator| and [=set/remove=] it from |issuedRequests|.
 
+                Note: An [=authenticator=] may return some indication of "the user cancelled the entire operation".
+                Conveyance of such an indication is client platform-specific.
```

Comment:
Ditto

Reply:
Yeah... maybe it could just be "It is unspecified how a user agent should convey this state to the user" ?

Reply:
yeah, how this state is annunciated is another aspect of this.